code/modules/mob/living/carbon/xenobiological/colours/colour_quantum.dm

```diff
@@ -0,0 +1,17 @@
+/decl/slime_colour/quantum
+	name = "quantum"
+	baby_icon =      'icons/mob/slimes/slime_baby_quantum.dmi'
+	adult_icon =     'icons/mob/slimes/slime_adult_quantum.dmi'
+	extract_icon =   'icons/mob/slimes/slime_extract_quantum.dmi'
+	reaction_sound = 'sound/effects/teleport.ogg'
+	reaction_strings = list(/decl/material/solid/metal/uranium = "Randomly teleports everything around the core.")
+
+/decl/slime_colour/quantum/handle_uranium_reaction(var/datum/reagents/holder)
+	var/list/turfs = list()
+	for(var/turf/T in orange(holder.my_atom,6))
+		turfs += T
+	for(var/atom/movable/a in viewers(holder.my_atom,2))
+		if(!a.simulated)
```

Comment:
maybe anchored check too, sounds like disaster waiting to happen (teleporting APCs etc)